petasus/scripts/sage2lib.py

```diff
@@ -24,7 +24,11 @@ def read_peptides(peptides, qvalue):
     polars.DataFrame
         The parsed and filtered peptides.
     """
-    peptide_df = pl.read_csv(peptides, separator="\t")
+    try:
+        peptide_df = pl.read_parquet(peptides)
+    except pl.exceptions.ComputeError:
```

Comment:
R: Can we add a check of the error to make sure the compute error comes from the parsing and not anywhere else?